From 348bb23908459d78c6fd839be2fbd436ef96c73d Mon Sep 17 00:00:00 2001
From: Lyon Wang <lyon.wang@nxp.com>
Date: Wed, 24 Oct 2018 15:00:07 +0800
Subject: [PATCH 08/75] MMFMWK-7852 v4l2 error log

- When set fmt field to V4L2_FIELD_ANY
  It will print error log, as it is not a critical issue
  Change it to GST_WARNING

Upstream-Status: Pending

Signed-off-by: Lyon Wang <lyon.wang@nxp.com>
(cherry picked from commit 03a5908b4f21aed7340afbf55f8949440909d78c)
Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 sys/v4l2/gstv4l2object.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys/v4l2/gstv4l2object.c b/sys/v4l2/gstv4l2object.c
index 5032da3..8ce970f 100644
--- a/sys/v4l2/gstv4l2object.c
+++ b/sys/v4l2/gstv4l2object.c
@@ -2040,7 +2040,7 @@ gst_v4l2_object_get_interlace_mode (enum v4l2_field field,
 {
   switch (field) {
     case V4L2_FIELD_ANY:
-      GST_ERROR
+      GST_WARNING
           ("Driver bug detected - check driver with v4l2-compliance from http://git.linuxtv.org/v4l-utils.git\n");
       return FALSE;
     case V4L2_FIELD_NONE:
-- 
2.43.0

